hedera-node/configuration/preprod/bootstrap.properties

```diff
@@ -2,4 +2,5 @@ bootstrap.throttleDefsJson.resource=throttles.json
 contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
```

Comment:
We should change the existing one and remove the newly added one.

Reply:
Thanks for catching that!